src/client/components/Meals/CreateMeal.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import Moment from 'moment';
+import OkModal from '../Helper/OkModal'
+
+const Createmeal = () => {
+    const [newMeal, setNewMeal] = useState();
+    const [title, setTitle] = useState("");
+    const date = Moment(new Date()).format('YYYY-MM-DD');
+    const [maxReservations, setMaxReservations] = useState();
+    const [description, setDescription] = useState("");
+    const [location, setLocation] = useState("");
+    const [when, setWhen] = useState();
+    const [price, setPrice] = useState();
```

Comment:
Again since you keep the state of title, description, etc... you don't have to also keep a newMeal object. Just creating it on submit to make the post request would be enough.

Reply:
Okay